### PR TITLE
Add and Remove task Functionality

### PR DESCRIPTION
**Pull request by Nicolaswg**
## Main point

> In this project we can see a basic functionality of adding, removing, and editing the task list inside the DOM with the localStorage and preserving the same data when refreshing the page.

See the LocalStorage information for interactions of the requirements 

### Interactions 

* > The add function can be done by clicking the plus button or hitting enter

* > The remove function splice the actual local storage and return the remaining elements 

* > The edit function edit the DOM description and update the local storage

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,10 +11,10 @@
     <div class="wrapp-list">
         <div class="header">
             <p>Today's To Do</p>
-            <i class="fas fa-sync"></i>
         </div>
         <div class="inputfield">
             <input type="text" placeholder="Add to your list ..." id="task">
+            <i class="fas fa-plus" id="btn-add"></i>
```
Comment (moise10r) [inline src/index.html:17]: - [ ] Great job so far you project work perfectly weel :clap:  whatever you should only display one button at once, the ` delete` button can only be displayed after making a click on the ` edit` button

Your:



Original:

**Review by moise10r — APPROVED** [review]
### Hi **@ Nicolaswg,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Good commit messages ✔️ 
- Your project woks as expected, keep rocking :rocket:
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (Nicolaswg) [reply src/index.html:17]: thanks for your review, I just want to know if this is 100 % require ?
just because the project works as respected.

**Review by Cathella — APPROVED** [review]
## 🟢 Status: Approved 🔥 

### Hi @Nicolaswg 👋🏻, 

Great job!, I really like your project and I think it looks great at this point 👍🏻.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**